public/main.js

```diff
@@ -0,0 +1,26 @@
+const randomNumber = Math.floor(Math.random() * (100) + 1)
+
+let counter = 0
+
+const guessNumber = (input) => {
+  let inputValue = input.value
+  console.log(randomNumber);
+  if (isNaN(inputValue) || inputValue < 1 || inputValue > 100) alert('Must be a number between 1 and 100')
+  if (inputValue == randomNumber) alert('You Win!')
+  if (inputValue > randomNumber && inputValue <= 100) {
```

Comment:
You have already confirmed that `inputValue` is <= 100 in your first conditional.